docs/store/starling.md

```diff
@@ -12,61 +12,92 @@ breadcrumb: 'Starling'
 <a href="https://starlingstorage.io" target="_blank"><img src="./images/starling.gif" alt="starling" /></a>
 </center>
 
-## Get started with Starling
+## Getting started
 
-1. Starling uses [Lotus](../get-started/lotus/README.md) to interact with the Filecoin network. [download, install and run a Lotus daemon](../get-started/lotus/installation.md):
+This guide will quickly set up Starling on your computer. For more information on Starling and how to use it, check out the [official Starling documentation](https://starlingstorage.io/).
```

Comment:
The official starling documentation did not work when I tried. Therefore it is not referenced here until the end, once the user has followed our instructions to avoid the temptation of following it. I don't know if the starling npm module is updated frequently, but some critical fixes had only landed on master and given the pace of development it is probably better to use master for the moment..

Reply:
```suggestion
This guide will quickly set up Starling on your computer.
```